scripts/Split_View_Figure.py

```diff
@@ -111,6 +111,7 @@ def create_figure_file(conn, figure_json):
     update = conn.getUpdateService()
     orig_file = conn.createOriginalFileFromFileObj(
         f, '', figure_name, file_size, mimetype="application/json")
+    f.close()
```

Comment:
A try/finally will be better otherwise the resource will never be closed if an error occurs while creating the image